system-int-test/test/cmr/system_int_test/search/granule_search_format_test.clj

```diff
@@ -38,6 +38,27 @@
     (ingest/create-provider {:provider-guid "provguid1" :provider-id "PROV1"})
     (ingest/create-provider {:provider-guid "provguid2" :provider-id "PROV2"})))
 
+(deftest simple-search-test-for-gran-echo-1.6.3-schema
+  (let [coll-7340 (d/ingest-concept-with-metadata-file "CMR-7340/Coll-CMR-7340.echo10"
+                                                  {:provider-id "PROV1"
+                                                   :concept-type :collection
+                                                   :native-id "echo10-collection"
+                                                   :format-key :echo10})
+        ;; This granule contains additional file added in the new schema
+        gran-7340 (d/ingest-concept-with-metadata-file "CMR-7340/Gran-CMR-7340.echo10"
+                                                     {:provider-id "PROV1"
+                                                      :concept-type :granule
+                                                      :native-id "echo10-granule"
+                                                      :format-key :echo10})]
+    (index/wait-until-indexed)
+    (let [params {:concept-id (:concept-id gran-7340)}
```

Comment:
I have seen other CMR developers complain about lets inside of lets, but I understand the equal dislike for doing _ (index….). If you think others will complain you may want to change this

Reply:
I disagree. We have many tests that are like that. It's difficult to NOT to use two lets because we have to wait till it's indexed.